### DISCUSSION TITLE
Auth Desktop - Support otpauth:// urls

### DISCUSSION THREAD

**Discussion by paulxb** [ente-io/ente#1285] [Enhancements]
I develop multi-factor solutions for websites and during testing I am always adding and deleting auth codes. It would be good if Ente Auth on Windows supported otpauth:// links so I can add new codes quickly by just clicking a link on the website.

References:
https://github.com/google/google-authenticator/wiki/Key-Uri-Format
https://www.ietf.org/archive/id/draft-linuxgemini-otpauth-uri-00.html

**Comment by ua741** [comment]
Related https://github.com/ente-io/ente/discussions/4300
